src/styles/breakpoints.ts

```diff
@@ -0,0 +1,48 @@
+import { Dimensions } from 'react-native'
+
+enum ScreenSize {
+  small = 'small',
+  medium = 'medium',
+  large = 'large',
+}
+
+const breakpoints = {
+  [ScreenSize.large]: {
+    width: 360,
+    height: 720,
+  },
+  [ScreenSize.medium]: {
+    width: 360,
+    height: 640,
+  },
+}
+
+interface StyleValues {
+  [ScreenSize.small]: number | string
+  [ScreenSize.medium]: number | string
+  [ScreenSize.large]: number | string
```

Comment:
Are the string values the ones listed above? I.e. 'small', 'medium', 'large'? Because if yes you could type is as `ScreenSize` instead of `string` to have stricter type checking.

Reply:
The string values are for spacing expressed in percentaces, i.e. "30%".

Reply:
Okay thanks, got it. My suggestion isn't valid then. Will mark as resolved.